conda_forge_tick/mamba_solver.py

```diff
@@ -560,6 +560,11 @@ def clean():
             subdirs=["osx-64", "osx-arm64"],
         )
     repodata.add_package(FakePackage("__win", "0"), subdirs=["win-64"])
+    repodata.add_package(FakePackage("__linux", "0"), subdirs=["linux-64"])
+    repodata.add_package(
+        FakePackage("__unix", "0"),
+        subdirs=["linux-64", "osx-64", "osx-arm64"],
```

Comment:
Do we need this to be
```suggestion
        subdirs=["linux-64", "linux-aarch64", "linux-ppc64le", "osx-64", "osx-arm64"],
```
?
(Probably not extremely important -- I'd assume only `conda-forge-pinning` is the only one for now: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/4bb0df15df4e0af677ddf6ab8add088ef5d5c0de/conda-forge.yml#L7-L8 )

Reply:
ack thank you!